source/fe/fe.cc

```diff
@@ -845,12 +845,13 @@ FiniteElement<dim, spacedim>::hp_constraints_are_implemented() const
 template <int dim, int spacedim>
 const FullMatrix<double> &
 FiniteElement<dim, spacedim>::constraints(
-  [[maybe_unused]] const internal::SubfaceCase<dim> &subface_case) const
+  const internal::SubfaceCase<dim> &subface_case) const
 {
   // TODO: the implementation makes the assumption that all faces have the
   // same number of dofs
   AssertDimension(this->n_unique_faces(), 1);
   [[maybe_unused]] const unsigned int face_no = 0;
+  (void)subface_case;
```

Comment:
Does `[[maybe_unused]]subface_case;` not work? Bummer.

Reply:
```
/src/source/fe/fe.cc:854:3: warning: attributes at the beginning of statement are ignored [-Wattributes]
  854 |   [[maybe_unused]]subface_case;
```

Reply:
yes, that works and that was my first draft. I don't particularly like it.